l10n_de_website_sale_accept_terms/readme.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
The file needs to be renamed README.rst

this should fix the warning and make runbot green.